src/main/java/ch/njol/skript/classes/data/SkriptClasses.java

```diff
@@ -675,6 +678,72 @@ public String toVariableNameString(VisualEffect e) {
 				.serializer(new YggdrasilSerializer<GameruleValue>())
 		);
 
+		Classes.registerClass(new ClassInfo<>(Config.class, "config")
+			.user("configs?")
+			.name("Config")
+			.description("A configuration (or code) loaded by Skript, such as the config.sk or aliases.",
+				"Configs can be reloaded or navigated to find options.")
+			.usage("")
+			.examples("the skript config")
+			.since("INSERT VERSION")
+			.parser(new Parser<Config>() {
+
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public @Nullable Config parse(String name, ParseContext context) {
+					return null;
+				}
+
+				@Override
+				public String toString(Config config, int flags) {
+					return this.toVariableNameString(config);
+				}
+
+				@Override
+				public String toVariableNameString(Config config) {
+					@Nullable File file = config.getFile();
+					if (file == null)
+						return config.getFileName();
+					return ExprScripts.FOLDER_PATH.relativize(file.toPath().toAbsolutePath()).toString();
```

Comment:
I'm not a huge fan of referencing an expression like this. I would prefer for you to just duplicate the field here.

Reply:
I think this has been moved in a PR since so I should be able to get rid of it :)